### PR TITLE
Partial port to new input system for simulation

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

Opening this to get some of the helpers I've created and a partial input system port into the repo. This doesn't yet bring us 100% to the new input system but does clean up some of the usage error logs while using the new input system.

The main remaining piece is figuring out the right way to handle our custom input module.

## Changes

- Part of #10360

### CODE REVIEW

File: Assets/TestMouse.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+using System.Collections;
```
Comment (keveleigh) [inline Assets/TestMouse.cs:1]: Need to remove this

**Comment by davidkline-ms** [comment]
Closing due to the deep integration of the legacy input system in MRTK v2. V3 is 100% based on the new input system.

**Pull request closed without merging**